lib/propolis/src/vcpu.rs

```diff
@@ -709,7 +726,7 @@ pub mod migrate {
                 self.intr_shadow as u64,
             )?;
 
-            let mut ents = [
+            let ents = [
```

Comment:
Since bypassing the conditional is more the outlier than the norm, I'm inclined to leave these here for now.  At some point, we'll bump the minimum vmm API version and unwrap it from the check.